community/kernel/src/main/java/org/neo4j/kernel/internal/Version.java

```diff
@@ -170,6 +149,20 @@ public static void main( String[] args )
     }
 
     static final String KERNEL_ARTIFACT_ID = "neo4j-kernel";
-    private static final Version KERNEL_VERSION = new Version( KERNEL_ARTIFACT_ID,
-            Version.class.getPackage().getImplementationVersion() );
+    private static final Version KERNEL_VERSION = new Version( KERNEL_ARTIFACT_ID, getMavenVersion() );
```

Comment:
ok, I'll make sure that information is still available. But the existing implementation only works once the whole project is in a jar-form. It does not work when running tests. Having the properties file ALWAYS works.